### PR TITLE
[prometheus-cloudwatch-exporter] configurable pod annotations

### PR DESCRIPTION
**Pull request by chelomontilla**
Signed-off-by: Chelo Montilla <chelo.montilla@lansweeper.com>

#### What this PR does / why we need it:
This PR allows to configure pod annotations in prometheus cloudwatch exporter chart.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-cloudwatch-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.10.0"
 description: A Helm chart for prometheus cloudwatch-exporter
 name: prometheus-cloudwatch-exporter
-version: 0.16.0
+version: 0.16.1
```
Comment (monotek) [inline charts/prometheus-cloudwatch-exporter/Chart.yaml:5]: new feature should raise minor version

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-cloudwatch-exporter/Chart.yaml (hunk 1 above)
Comment (chelomontilla) [reply charts/prometheus-cloudwatch-exporter/Chart.yaml:5]: ok

**Review by chelomontilla — COMMENTED** [review]
thanks

**Comment by chelomontilla** [comment]
could someone please merge this ? the pr was auto closed

**Comment by monotek** [comment]
You've still raising the patch version. 
Please raise minor version: https://semver.org/lang/de/

**Review by monotek — COMMENTED** [review]
You've still raising the patch version.
Please raise minor version: https://semver.org/lang/de/

**Review by monotek — APPROVED** [review]

**Pull request merged**